iOS/AppboyReactBridge/AppboyReactBridge/AppboyReactBridge.m

```diff
@@ -289,7 +289,7 @@ - (ABKCardCategory)getCardCategoryForString:(NSString *)category {
 
 RCT_EXPORT_METHOD(launchFeedback) {
   RCTLogInfo(@"launchFeedback called");
-  ABKFeedbackViewControllerModalContext *feedbackModal = [[ABKFeedbackViewControllerModalContext alloc] init];
```

Comment:
Per https://github.com/Appboy/appboy-ios-sdk/releases/tag/2.31.0, ABKFeedbackViewControllerModalContext was actually renamed to ABKModalFeedbackViewController.